src/main/java/core/basesyntax/service/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final String INPUT_FILE = "fruit_shop_input.csv";
```

Comment:
Remove it. Pass file as a input parameter. Also use resources directory for the saving files